src/software/ai/hl/stp/play/shoot_or_pass_play.cpp

```diff
@@ -86,12 +97,41 @@ void ShootOrPassPlay::getNextTactics(TacticCoroutine::push_type &yield,
         auto receiver = std::make_shared<ReceiverTactic>(
             world.field(), world.friendlyTeam(), world.enemyTeam(), pass, world.ball(),
             false);
+
+        auto pass_eval = pass_generator.generatePassEvaluation(world);
+
+        Zones cherry_pick_region_1 = {pass_eval.getBestZonesToCherryPick(
+            world.field(), best_pass_and_score_so_far.pass.receiverPoint())[0]};
+        Zones cherry_pick_region_2 = {pass_eval.getBestZonesToCherryPick(
+            world.field(), best_pass_and_score_so_far.pass.receiverPoint())[1]};
```

Comment:
I'd add a call like:
```
auto best_zones_to_cherry_pick = pass_eval.getBest....
```
before to avoid duplicate function calls

Reply:
added